Source/WebCore/platform/text/cocoa/LocalizedDateCache.mm

```diff
@@ -122,7 +122,8 @@ static void _localeChanged(CFNotificationCenterRef, void*, CFStringRef, const vo
         [dateFormatter setDateStyle:NSDateFormatterNoStyle];
         break;
     case DateComponentsType::Week:
-        ASSERT_NOT_REACHED();
+        [dateFormatter setTimeZone:[NSTimeZone timeZoneForSecondsFromGMT:0]];
```

Comment:
I know you're just following the pattern here, but it looks like we make this same call for all cases... maybe pull it out?